packages/jest-config/src/Descriptions.ts

```diff
@@ -7,6 +7,7 @@
 
 import type {Config} from '@jest/types';
 
+// TODO should we had the description of id that was previously name
 const descriptions: {[key in keyof Config.InitialOptions]: string} = {
```

Comment:
Should we have a description of what the id does? @SimenB 
And should it be added in the docs? https://jestjs.io/docs/en/cli
If so what would be the correct description?